e2e/scripts/st_markdown.py

```diff
@@ -88,3 +88,16 @@
 - :rainbow[rainbow]
 """
 )
+
+st.markdown(
```

Comment:
Might also be good to add some additional tests here with a bit more complexity, e.g.: A multiline string with background color, usage of other markdown instructions inside the text with the background color

Reply:
> usage of other markdown instructions inside the text with the background color

I can do that 👍 

> A multiline string with background color

Neither colored text nor background colors for text support multiline strings though..
i.e. a separate `:color[text]` or `:color-background[text]` need to be called on each line. Are you ok with that?

Doesn't work:

```py
st.markdown(
    """
    :blue[
    **Bold text within blue**\n\n
    *Italic text within blue*\n\n
    [Link](http://example.com) within blue
    ]
    """
)

st.markdown(
    """
    :blue-background[
    **Bold text within blue background**\n\n
    *Italic text within blue background*\n\n
    [Link](http://example.com) within blue background
    ]
    """
)
```

<img width="370" alt="image" src="https://github.com/streamlit/streamlit/assets/20672874/0e6a37bb-5fae-4fa1-a2b2-1ab119b95021">

Reply:
@LukasMasuch everything besides multiline testing is done :-)

Reply:
Added on f491199